src/containers/HomePage.js

```diff
@@ -4,31 +4,91 @@ import Menu from 'components/common/Menu';
 import LogoutButton from 'components/user/LogoutButton';
 import SimpleMap from 'components/common/maps/Map';
 import Welcome from 'components/user/Welcome';
+import CreateTargetForm from 'components/user/CreateTargetForm';
 import { array, func } from 'prop-types';
 
-import { loadTargets } from '../actions/targetActions';
+import { loadTargets, addTarget } from '../actions/targetActions';
 import { loadTopics } from '../actions/topicActions';
+import { FormattedMessage } from 'react-intl';
 
 class HomePage extends PureComponent {
+  constructor() {
+    super();
+    this.state = {
+      targetPosition: {},
+      isCreatingNewTarget: false,
+    };
+    this.onClickMap = this.onClickMap.bind(this);
+    this.handleCreateTarget = this.handleCreateTarget.bind(this);
+  }
+
   componentDidMount() {
     this.props.loadTargets();
     this.props.loadTopics();
   }
 
+  onClickMap = ({ x, y, lat, lng, event }) => {
+    console.log(x, y, lat, lng, event);
+    const targetPosition = {
+      lat,
+      lng
+    };
+    this.setState({ targetPosition });
+    this.setState({ isCreatingNewTarget: true });
+  }
+
+  handleCreateTarget(data) {
```

Comment:
I would change this function to an arrow function to remove the bind line

Reply:
It´s done!!